tests/cart/test_add_item_to_cart_negative.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+from allure import description, step, title, feature
+from hamcrest import assert_that, has_length, has_key
+
+from configs import id_product_not_exist_in_BD_for_adding_to_cart
```

Comment:
`id_product_not_exist_in_BD_for_adding_to_cart` is a test data, isn't it?
Would you mind moving it to a separate `data.py` (feel free to choose any name) and commit this file? At the moment I can't run tests from this repo locally, because dependencies like this are unresolved

Reply:
Resolve